Documentation/contributing/making-changes.rst

```diff
@@ -202,6 +202,56 @@ squash before submitting the Pull Request:
 
        $ git checkout my-branch
 
+#. Fetch the upstream code
+
+    .. code-block:: bash
+
+       $ git fetch upstream
+
+#. Rebase onto the upstream code
+
+   * Your favorite text editor will start and shows the commands for the interactive rebasing.
+   * Change the "pick" to "squash" for all the commits you want to squash.
+   * For the series of commits to squash, keep the first one with "pick".
+   * Once you save and exist your editor will start again to merge all the commit messages.
+   * Add a commit message that best describes your change then save and exit.
+
+    .. code-block:: bash
+
+       $ git rebase upstream/master
+
+#. Push to your remote
+
+   This needs to a force push with ``-f``.
```

Comment:
Force is not needed here since this would be the first push which creates the remote branch. The force will only be needed if you squash, for example, or if during reviews you are requested to rebase.

Reply:
@v01d I do this flow all the time, and yes ``-f`` is needed, since you have an existing branch on the server with the same name.

Reply:
Ah, I didn't see the previous git push to the fork, sorry.